README.md

```diff
@@ -110,103 +109,103 @@ manager       cord-kafka-0                               1/1     Running   1
 manager       cord-kafka-zookeeper-0                     1/1     Running   0          5h26m   192.168.253.129   device-manager   <none>           <none>
 ```
 
-## Go package Installation
-The go command will use in building the Device Manager source codes.
-```shell
-#> make go-install
-```
+## Install Go package
+The following command will install GoLang compile.
 
 ```shell
-#> . ~/.bashrc      #Take effect Go environment variables
+$> make go-install
+$> . ~/.bashrc      #Take effect Go environment variables
 ```
 
-## Protobuf package installation and download the necessary GO libraries.
-The protoc command will use in building the .proto file.
+## Install Protobuf
+The following command shall install protoc.  protoc is use to build the .proto file.
+
 ```shell
-#> make prereq
+$> make prereq
 ```
 
-## Download and build Device management docker image
-The images of device management will be downloaded to the host, And build those source files.
+## Build Device Manager docker image
+The following command will build the Device Manager docker image.
 
-The following by this command to build the Device Management docker image
 ```shell
-#> make build-dm
+$> make build-dm
 ```
 
-## Install Device management Pod.
-The device management would follow commands to bring up the Pod.
+## Bring up Device Persistent Volume (dpv)
+Bring up the Device Persistent Volume as a Helm chart (device-pv).
 
-Bring up the Device Persistent Volume first (Default: /var/devices_data) . The device data file could store in the host platform.
 ```shell
-#> make dpv
+$> make dpv
 ```
 
-Displaying the device persisent volume status
+Display the device persistent volume status
+
 ```shell
-#> helm ls
+$> helm ls
 ```
 NAME            REVISION        UPDATED                         STATUS          CHART                           APP VERSION     NAMESPACE
 cord-kafka      1               Wed Nov 11 18:28:40 2020        DEPLOYED        kafka-0.13.3                    5.0.1           default
 devices-pv      1               Wed Nov 11 20:47:46 2020        DEPLOYED        local-directory-0.1.0-dev0                      default
 
 
-Bring up the Device-management Pod
+Bring up the Device Manager Kubenetes Pod
 ```shell
-#> make dm
+$> make dm
 ```
 
-After bring up the Pods, you could use the command to check the status of device management Pod.
+After bring up the device manager pod, the command can be used to check the status of Device Manager Pod.
```

Comment:
Device Manager instead of device manager

Reply:
Fixed.  Also changed 'bring' to 'bringing'